src/lib.rs

```diff
@@ -23,12 +23,13 @@ pub const JUNO_1: ChainInfo = ChainInfo {
 pub const IBC_CLIENT_ID: &str = "abstract:ibc-client";
 
 // @feedback: it would be really nice to be able to query a module directly from the account
+// @uno-feedback: Why not use abstract-client `Application` helper?
```

Comment:
Didn’t know we could use it for the ibc-client at this point. Would be great to have examples on using it with it on the docs to demonstrate its use.

Reply:
I think it's still worth adding `query_module(module_id, &msg)` on `Account`, do you agree? It's possible that someone else abstract module(most likely standalone) won't have required traits to get `Application` object